linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/bash/shared.sh

```diff
@@ -1,7 +1,7 @@
 # platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
 
-# Find which files have incorrect hash (not in /etc, because there are all system related config. files) and then get files names
-files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | awk '{print $NF}' )"
+# Find which files have incorrect hash and then get files names
+files_with_incorrect_hash="$(rpm -Va {{{ "" if product == "rhel6" else "--noconfig" }}} | grep -E '^..5' | awk '{print $NF}' )"
```

Comment:
Can this be simpler?
Would `{{{ "--noconfig" if product != "rhel6" }}}` work?

Reply:
I guess it can. Will update accordingly. Thanks.

Reply:
Updated by amending last commit.